### DISCUSSION TITLE
What's the different between net.peer.ip and http.client_ip in a Span

### DISCUSSION THREAD

**Discussion by lmssy** [open-telemetry/opentelemetry-java-instrumentation#3963] [Q&A] [answered]
I have an application provide http api, in a span with kind SPAN_KIND_SERVER there are **net.peer.ip** and **http.client_ip** in attributes with different ip values.

So what's the different of these 2 attributes? I think they should be a same ip which indicate the client who call my api.

**Comment by iNikem** [comment, marked as answer]
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#http-server-semantic-conventions actually explains this:

> [http.client_ip] is not necessarily the same as net.peer.ip, which would identify the network-level peer, which may be a proxy

**Reply by lmssy** [reply]
Ah I see, Thanks a lot!
